web_monitoring/db.py

```diff
@@ -142,13 +143,17 @@ class method :meth:`Client.from_env`.
     url : string, optional
         Default is ``https://api.monitoring.envirodatagov.org``.
     """
-    def __init__(self, email, password, url=DEFAULT_URL):
+    def __init__(self, email, password, url=DEFAULT_URL, timeout=DEFAULT_TIMEOUT):
         self._api_url = f'{url}/api/v0'
         self._base_url = url
         self._session = requests.Session()
         self._session.auth = (email, password)
         self._session.headers.update({'accept': 'application/json'})
 
+        if timeout is not None and timeout <= 0:
+            raise ValueError(timeout)
```

Comment:
This should have a more descriptive message. Also, I’m thinking `0` should probably be OK — it should imply “no timeout,” i.e. same as `None`. That is a pretty typical interpretation of `0` in a lot of timeout-related systems.